Sources/Controllers/2.0/OAMapSettingsOnlineSourcesScreen.mm

```diff
@@ -55,6 +63,11 @@ - (id) initWithTable:(UITableView *)tableView viewController:(OAMapSettingsViewC
         CGSize btnSize = [OAUtilities calculateTextBounds:OALocalizedString(@"shared_string_done") width:kMaxDoneWidth font:vwController.okButton.titleLabel.font];
         [vwController.okButton setConstant:@"buttonWidth" constant:btnSize.width + 32.];
         
+        if ([param isEqualToString:@"overlay"])
```

Comment:
Since param introduced, _mapSettingType must be defined correctly here for all cases. Now if param is empty - _mapSettingType = underlay which is wrong.